### PR TITLE
FIX: missing i18n

### PR DESCRIPTION
**Pull request by limpbrains**
Fixing #2301

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/0u5rr4wrpeaupdwgncp72jkbvg

[download apk](http://filestorage.bluewallet.io:1488/1608576793.apk)

File: BlueComponents.js [hunk 1]
```diff
@@ -2072,7 +2072,7 @@ export class BlueBitcoinAmount extends Component {
                 maxLength={this.maxLength()}
                 ref={textInput => (this.textInput = textInput)}
                 editable={!this.props.isLoading && !this.props.disabled}
-                value={parseFloat(amount) >= 0 || amount === BitcoinUnit.MAX ? amount : undefined}
+                value={amount === BitcoinUnit.MAX ? loc.units.MAX : parseFloat(amount) >= 0 ? amount : undefined}
```
Comment (Overtorment) [inline BlueComponents.js:2075]: not sure about this one. I think we have MAX as a reserve constant by which we determine if user wants to send all. so in other languages sendMax might just break. needs verifying

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: BlueComponents.js (hunk 1 above)
Comment (limpbrains) [reply BlueComponents.js:2075]: This line doesn't change internal `state.amount`, it changes its representation in TextInput. I've checked, works fine even with russian МАКС in `loc.units.MAX`

**Review by marcosrdz — CHANGES_REQUESTED** [review]
Lint

**Comment by limpbrains** [comment]
@marcosrdz synced with master fixed the lint

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wduyph1a9h450xgwmp4bw6mqww

[download apk](http://filestorage.bluewallet.io:1488/1608836847.apk)

**Review by marcosrdz — APPROVED** [review]

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**